pkg/kubernetes/persistentvolumeclaim/v1alpha1/persistentvolumeclaim.go

```diff
@@ -180,3 +183,94 @@ func (b *ListBuilder) WithFilter(pred ...Predicate) *ListBuilder {
 	b.filters = append(b.filters, pred...)
 	return b
 }
+
+// NewPVC returns a PVC instance
+func NewPVC() *PVC {
```

Comment:
This is not what idiomatic maya follows?
Why are we trying to deviate from our standard?

Reply:
Refactored the code